docs/reference/commandline/service_create.md

```diff
@@ -245,8 +245,8 @@ metadata](https://docs.docker.com/engine/userguide/labels-custom-metadata/).
 
 ### Add bind-mounts or volumes
 
-Docker supports two different kinds of mounts, which allow containers to read to
-or write from files or directories on other containers or the host operating
+Docker supports two different kinds of mounts, which allow containers to read
+from or write to files or directories on other containers or the host operating
```

Comment:
Thanks! 

The `on other containers` part is incorrect as well; mounts are always mounted from the host to the container; if two containers share the same host path for a mount, docker is mounting that path from the host, not the other container's filesystem.

Can you update your PR and remove the `on other containers`?